Releases/v1.4.0 rc1 (#176)
fyne-io/fyne-cross: README.md

go install github.com/fyne-io/fyne-cross@latest
```

> To install a fyne-cross with kubernetes engine:

Reviewer: ```suggestion
To install a fyne-cross with kubernetes engine support:
```

Use "fyne-cross <command> -help" for more information about a command.
```

### Apple Arm Silicon special

> On Apple Arm64 architecture, the container have to run in a vm. This vm put some constraint on getting fyne-cross to work in that setup.

Reviewer: ```suggestion
On Apple Arm64 architecture, the container has to run in a vm. This vm puts some constraints on getting fyne-cross to work with that setup.
```

Author: Oh, forgot we had some stuff in the README regarding Apple Arm64. I have clarified that now as it was inaccurate.

## podman on Mac Mini M1

> You need to create a `podman machine` that has a short name as the access path on Mac can not be over 104 and that limit can be reached quickly when the machine name is to long. Additionally you need to specify in the machine the path that you might need during your build. An example of possible machine to use:

Reviewer: The first sentence is very long. I think you ought to split into two. I am also not entirely sure what the last one is trying to say. Can you make it a bit clearer?
